app/src/main/java/org/mozilla/fenix/library/history/HistorySearchController.kt

```diff
@@ -19,6 +21,7 @@ interface HistorySearchController {
 
 class HistorySearchDialogController(
     private val activity: HomeActivity,
+    private val metrics: MetricController? = null,
```

Comment:
why is metrics optional here?

Reply:
This is in-case us or anyone wants to create a `HistorySearchDialogController` then can opt out of sending metrics.  You're probably right that I shouldn't worry about that.  I'll remove the null default.  Thanks